Tests/Clappr_Tests/PlayerTests.swift

```diff
@@ -236,7 +236,7 @@ class PlayerTests: QuickSpec {
                         Player.register(plugins: [LoggerPlugin.self])
                         player = Player(options: options)
 
-                        let loggerPlugin = player.core?.plugins.first { $0 is LoggerPlugin }
+                        let loggerPlugin = player.getPlugin(name: LoggerPlugin.name)
```

Comment:
I was thinking about adding a test to check if it can return the correct plugin if the type of the plugin is `container`, but I think I'm being too picky. I'll approve it and let you decide if you create this test or not ;)